config/context/spring-context.xml

```diff
@@ -126,9 +126,16 @@
     <constructor-arg name="keyPair" ref="brvsAccountKeyPair"/>
     <constructor-arg name="irohaAPI" ref="irohaAPI"/>
   </bean>
+  <bean id="billingRule" class="iroha.validation.rules.impl.billing.BillingRule">
+    <constructor-arg name="getBillingURL" value="http://localhost:8080/cache/get/billing"/>
```

Comment:
Is it possible to set this value with environment variable?

Reply:
Not yet, we'll create a separate task for env processing

Reply:
It seems I misunderstood the comment, I'll put the value in the `.properties` so allowing some environment variable to appear. Thank you for spotting!